onnxruntime/core/providers/shared_library/provider_interfaces.h

```diff
@@ -543,35 +544,35 @@ struct CPUIDInfo {
   bool HasAVX2() const { return g_host->CPUIDInfo__HasAVX2(this); }
   bool HasAVX512f() const { return g_host->CPUIDInfo__HasAVX512f(this); }
 
-PROVIDER_DISALLOW_ALL(CPUIDInfo)
+  PROVIDER_DISALLOW_ALL(CPUIDInfo)
```

Comment:
nit: formatting

Reply:
Looks like clang format fixed it and this is correct now? (I might be misunderstanding it)